databricks-sdk-java/src/main/java/com/databricks/sdk/core/oauth/OAuthClient.java

```diff
@@ -116,6 +120,9 @@ private OAuthClient(Builder b) throws IOException {
     List<String> scopes = b.scopes;
     if (scopes == null) {
       scopes = Arrays.asList("offline_access", "clusters", "sql");
+    } else if (!scopes.contains("offline_access")) {
+      scopes = new ArrayList<>(scopes);
+      scopes.add("offline_access");
```

Comment:
Why should we add this always?

Reply:
Removed. Added to support refresh tokens, but clients may intentionally avoid this for short lived token use-case.